opencti-platform/opencti-graphql/src/database/data-initialization.js

```diff
@@ -70,16 +70,16 @@ export const CAPABILITIES = [
   KNOWLEDGE_CAPABILITIES,
   {
     name: 'EXPLORE',
-    description: 'Access Dashboards and investigations',
+    description: 'Access dashboards and investigations',
     attribute_order: 1000,
     dependencies: [
       {
         name: 'EXUPDATE',
-        description: 'Create / Update Dashboards and investigations',
+        description: 'Create / Update dashboards and investigations',
         attribute_order: 1100,
         dependencies: [
-          { name: 'EXDELETE', description: 'Delete Dashboards and investigations', attribute_order: 1200 },
-          { name: 'PUBLISH', description: 'Manage Public Dashboards', attribute_order: 1300 },
+          { name: 'EXDELETE', description: 'Delete dashboards and investigations', attribute_order: 1200 },
+          { name: 'PUBLISH', description: 'Manage public dashboards', attribute_order: 1300 },
```

Comment:
these description changes are not changed in the migration, couldn't we change them too for existing platforms ?

Reply:
The initial problem was to harmonise the use of capitals. As these changes have no impact on the way things work, it was decided with Adrien to fix this with a translation for existing platforms, and an update of the initial descriptions for new platforms. That's why there have to be 2 translations.